src/store/modules/product/actions.ts

```diff
@@ -226,8 +226,6 @@ const actions: ActionTree<ProductState, RootState> = {
       return product
     }
 
-    emitter.emit("presentLoader")
```

Comment:
We should also remove the dismissLoader code from here - https://github.com/hotwax/preorder/pull/209/files#diff-6768563d69ee4e12868366976dd54741f31c401ac52dd177f27b36d6ae1dd7c6R261